src/struct/HTTP.ts

```diff
@@ -21,6 +21,128 @@ export class HTTP {
 			});
 		});
 
+		this.app.get("/imageApi", async (req, res) => {
+			const { xp, xpToLevel, level, avatarURL, tag, status } = req.query;
+			const { createCanvas, loadImage, registerFont } = require("canvas");
+			const post = {
+				nickname: tag,
+				rank: level,
+				xp: xp,
+				toLevel: xpToLevel,
+				avatar: avatarURL
+			};
+
+			const width = 840;
+			const height = 214;
+
+			const canvas = createCanvas(width, height);
+			const context = canvas.getContext("2d");
+			registerFont(__dirname + '/../assets/fonts/Poppins-Bold.ttf', { family: 'Poppins Bold' });
+			registerFont(__dirname + '/../assets/fonts/Poppins-Light.ttf', { family: 'Poppins Light' });
+			registerFont(__dirname + '/../assets/fonts/Poppins-Regular.ttf', { family: 'Poppins Regular' });
```

Comment:
Also instead of registering fonts every time a request comes you should load them at the very top of the file once. We don't need to load fonts every time a request comes.